File: PHP/UberVan.php
<?php
require_once('car.php');
class UberVan extends Car {
    public $typeCarAcceted;
    public $seatsMaterial;

    public fuction __construct($license,$driver,$typeCarAcceted,$seatsMaterial){
        parent::__construct($license,$driver);
        $this->typeCarAcceted = $typeCarAcceted;
        $this->seatsMaterial = $seatsMaterial;
    }
}
?>

File: Python/UberVan.py
from car import Car 

class UberVan(Car):
    typeCarAcceted = []
    seatsMaterial = []
    
    def __init__(self,license, driver, typeCarAcceted, seatsMaterial):
        super().__init__(license, driver)
        self.typeCarAcceted = typeCarAcceted
        self.seatsMaterial = seatsMaterial